README.md

```diff
@@ -55,6 +55,14 @@ Would you like to find the tutorials in other languages? You can find all the tr
 >
 > 👉 [click here to access the tutorial](https://colab.research.google.com/github/huggingface/education-toolkit/blob/main/03_getting-started-with-transformers.ipynb)
 
+## **Our Teaching Guide: A Tour Through The 🤗 Hub & Gradio**
+
+In this video, Nate and Lewis summarize the importance of transformers and transfer learning and talk about open science efforts and tools to enable people to work collaboratively in their Machine Learning projects.
```

Comment:
```suggestion
In this video, Nate and Lewis give you a guided tour of Transformers and transfer learning, along with an overview of Hugging Face's open science efforts and tools that enable people to work collaboratively in their Machine Learning projects.
```

Reply:
I do not know if I succeeded in embedding the video correctly. Can you confirm that it works?